src/probnum/diffeq/ode/ivp_examples.py

```diff
@@ -0,0 +1,464 @@
+        or Normal (noisy) with :math:`3`-dimensional mean vector and
+        :math:`3 \times 3`-dimensional covariance matrix.
+        To replicate "classical" initial values use the Constant distribution.
+    Returns
+    -------
+    IVP
+        IVP object describing the rigid body dynamics IVP with the prescribed
+        configuration.
+    """
+
+    def rhs(t, y):
+        return rigidbody_rhs(t, y)
+
+    def jac(t, y):
+        return rigidbody_jac(t, y)
+
+    return IVP(timespan, initrv, rhs, jac=jac)
+
+
+def rigidbody_rhs(t, y):
+    y1, y2, y3 = y
+    return np.array([y2 * y3, -y1 * y3, -0.51 * y1 * y2])
+
+
+def rigidbody_jac(t, y):
+    y1, y2, y3 = y
+    return np.array([[0.0, y3, y2], [-y3, 0.0, -y1], [-0.51 * y2, -0.51 * y1, 0.0]])
+
+
+def vanderpol(timespan, initrv, params=0.1):
+    r"""Initial value problem (IVP) based on the Van der Pol Oscillator.
+
+    The Van der Pol Oscillator is defined through
+
+    .. math::
+
+        f(t, y) =
+        \begin{pmatrix}
+            y_2 \\
+            \mu \cdot (1 - y_1^2)y_2 - y_1
+        \end{pmatrix}
+
+    for a constant parameter  :math:`\mu`.
+    :math:`\mu` determines the stiffness of the problem, where
+    the larger :math:`\mu` is chosen, the more stiff the problem becomes.
+    Default is :math:`\mu = 0.1`.
+
+    Parameters
+    ----------
+    timespan : (float, float)
+        Time span of IVP.
+    initrv : RandomVariable,
+        *(shape=(2, ))* -- Vector-valued RandomVariable that describes the belief
+        over the initial value. Usually its distribution is Constant (noise-free)
+        or Normal (noisy) with :math:`2`-dimensional mean vector and
+        :math:`2 \times 2`-dimensional covariance matrix.
+        To replicate "classical" initial values use the Constant distribution.
+    params : (float), optional
+        Parameter :math:`\mu` for the Van der Pol Equations
+        Default is :math:`\mu=0.1`.
+
+    Returns
+    -------
+    IVP
+        IVP object describing the Van der Pol Oscillator IVP with the prescribed
+        configuration.
+    """
+
+    def rhs(t, y):
+        return vanderpol_rhs(t, y, params)
+
+    def jac(t, y):
+        return vanderpol_jac(t, y, params)
+
+    return IVP(timespan, initrv, rhs, jac=jac)
+
+
+def vanderpol_rhs(t, y, params):
+    y1, y2 = y
+    if isinstance(params, float):
+        mu = params
+    else:
+        (mu,) = params
+
+    return np.array([y2, mu * (1.0 - y1 ** 2) * y2 - y1])
+
+
+def vanderpol_jac(t, y, params):
+    y1, y2 = y
+    if isinstance(params, float):
+        mu = params
+    else:
+        (mu,) = params
+
+    return np.array([[0.0, 1.0], [-2.0 * mu * y2 * y1 - 1.0, mu * (1.0 - y1 ** 2)]])
+
+
+def threebody(timespan, initrv, params=0.012277471):
+    r"""Initial value problem (IVP) based on a three-body problem.
+
+    The three-body problem is defined as follows:
+    Let the initial conditions be :math:`y = (y_1, y_2, \dot{y}_1, \dot{y}_2)^T`.
```

Comment:
Can you also write that the threebody problem is turned into a first order linearisation in this function? IN the future we probably wont do this anymore.

Reply:
sorry for the mixed signals, this above comment is what happens if one "just does a quick review" in between two things... A first order linearisation of an ODE is bogus. I meant a first order ODE... Sorry!

Reply:
is this still relevant or resolved?

Reply:
maybe also in van der pol